staging/src/k8s.io/component-base/logs/json/klog_test.go

```diff
@@ -149,57 +163,63 @@ func TestKlogIntegration(t *testing.T) {
 			fun: func() {
 				klog.Error("test ", 1)
 			},
-			format: `{"ts":%f,"caller":"json/klog_test.go:%d","msg":"test 1\n","v":0}`,
```

Comment:
is it expected that these no longer output `"v":0`?

Reply:
Yes, verbosity for error didn't have any semantic meaning. It was always supposed to be equal zero, but we decided to drop it in zapr.

Reply:
Yes. Only info messages have a verbosity level and are subjected to filtering by verbosity. Error messages are always printed and therefore conceptually don't have verbosity. In other words, `V(3).Error` is the same as `V(0).Error`.